accord-core/src/test/java/accord/impl/list/ListStore.java

```diff
@@ -512,6 +513,8 @@ public static AsyncChain<SyncPoint<Ranges>> coordinate(Node node, long minEpoch,
             Await coordinate = new Await(node, minEpoch, sp);
             coordinate.start();
             return coordinate.recover(t -> {
+                if (t.getClass() == SyncPointErased.class)
+                    return AsyncChains.success(null);
```

Comment:
this code is consistent with the rest of the function, so its fine... but relooking at it `return AsyncChains.success(null)` feels wrong as the only caller doesn't handle that case!  Its there for debug logging, so if we actually log we will hit a NPE I think...

nothing for you to change as you are consistent... but a bad access will lead to a NPE if we hit this or `Redundant`